src/app/types.ts

```diff
@@ -0,0 +1,38 @@
+/**
```

Comment:
If you will name this file as ***.types.ts it would be handled automatically by TS
So you would need to import any file with types in this way